### PR TITLE
r-survival: add new pkgs

### PR DESCRIPTION
**Pull request by shun2wang**
## Involved packages

 - r-survival

## Involved issue

Close #

## Details

- [ ] Tested in the local machine (largest 16G RAM) and it is passed without any issue
- [x] Provide New Package
- [ ] Fix the Packages
  - [ ] PKGBUILD
  - [ ] lilac.yaml
  - [ ] lilac.py
- [ ] Would like to continue to work with us

### CODE REVIEW

**Comment by shun2wang** [comment]
I got a error report in email:

```
lilac.yaml of package r-ggsurvfit specifies repo_depends, but the (direct or indirect) dependency [Dependency(pkgdir=PosixPath('/usr/share/lilac/Packages/BioArchLinux/r-survival'), pkgname='r-survival')] is not in this repository.
```
and as we talked yesterday we should add it explicit.

File: BioArchLinux/r-survival/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Maintainer: Shun Wang <shuonwang at gmail dot com>
+# Contributor: Shun Wang <shuonwang@gmail.com>
+
+_pkgname=survival
+_pkgver=3.8-3
+pkgname=r-${_pkgname,,}
+pkgver=${_pkgver//-/.}
+pkgrel=1
+pkgdesc="Contains the core survival analysis routines, including definition of Surv objects, Kaplan-Meier and Aalen-Johansen (multi-state) curves, Cox models, and parametric accelerated failure time models."
```
Comment (shun2wang) [inline BioArchLinux/r-survival/PKGBUILD:9]: oh, I still use detail descriptions of r-pkg...this should not build for now?

Comment (sukanka) [reply BioArchLinux/r-survival/PKGBUILD:9]: You don't need to add `r-survival`, it's provided by `r`
```
➜  ~ yay -Q r-survival
r 4.4.2-1
```

Comment (sukanka) [reply BioArchLinux/r-survival/PKGBUILD:9]: The following are provided by `r` (see `Provides`), and  thus should not be submitted to this repo and AUR
```
➜  ~ yay -Qi r    
Name            : r
Version         : 4.4.2-1
Description     : Language and environment for statistical computing and graphics
Architecture    : x86_64
URL             : https://www.r-project.org/
Licenses        : GPL
Groups          : None
Provides        : r-kernsmooth=2.23.24  r-mass=7.3.61  r-matrix=1.7.1  r-base=4.4.2  r-boot=1.3.31  r-class=7.3.22  r-cluster=2.1.6  r-codetools=0.2.20  r-compiler=4.4.2  r-datasets=4.4.2  r-foreign=0.8.87
                  r-grdevices=4.4.2  r-graphics=4.4.2  r-grid=4.4.2  r-lattice=0.22.6  r-methods=4.4.2  r-mgcv=1.9.1  r-nlme=3.1.166  r-nnet=7.3.19  r-parallel=4.4.2  r-rpart=4.1.23  r-spatial=7.3.17
                  r-splines=4.4.2  r-stats=4.4.2  r-stats4=4.4.2  r-survival=3.7.0  r-tcltk=4.4.2  r-tools=4.4.2  r-translations=4.4.2  r-utils=4.4.2
Depends On      : bash  blas  bzip2  cairo  curl  gcc-libs  glib2  glibc  icu  lapack  libjpeg-turbo  libpng  libtiff  libtirpc  libx11  libxmu  libxt  pango  pcre2  readline  tk  unzip  which  xz  zip  zlib
Optional Deps   : blas-openblas: faster linear algebra [installed]
                  gcc-fortran: needed to compile some CRAN packages [installed]
                  texlive-latex: latex sty files [installed]
Required By     : None
Optional For    : graphviz
Conflicts With  : None
Replaces        : None
Installed Size  : 73.49 MiB
Packager        : Antonio Rojas <arojas@archlinux.org>
Build Date      : Fri Nov 1 03:00:50 2024
Install Date    : Fri Nov 1 20:27:56 2024
Install Reason  : Explicitly installed
Install Script  : No
Validated By    : Signature
```

**Comment by sukanka** [comment]
> I got a error report in email:
> 
> ```
> lilac.yaml of package r-ggsurvfit specifies repo_depends, but the (direct or indirect) dependency [Dependency(pkgdir=PosixPath('/usr/share/lilac/Packages/BioArchLinux/r-survival'), pkgname='r-survival')] is not in this repository.
> ```
> 
> and as we talked yesterday so we should add it explicit.

for `r-ggsurfit`, removing  `- r-survival` from `lilac.yaml` should work.

**Comment by shun2wang** [comment]
Oh, OK then I'll delete it.

**Pull request closed without merging**